Gemfile

```diff
@@ -62,7 +63,7 @@ gem "sunspot_rails", "~> 2.7"
 gem "sunspot_solr", "~> 2.7"
 gem "terser", "~> 1.2" # Ruby wrapper for UglifyJS JavaScript compressor
 gem "thredded", git: "https://github.com/thredded/thredded.git",
-                ref: "1340e913affd1af5fcc060fbccd271184ece9a6a"
+                ref: "94fbf890a984527bab399c8c7f1eec7af7006843"
```

Comment:
what do you think of going directly to [566100f6](https://github.com/thredded/thredded/commits/main/)? This would mean just 2 more commits, while with the current changing we are already bringing in 7 more (that cold break thredded 🙈).

Reply:
The advantage would be that we'd have already Rails 8.0 support integrated into thredded ;)

Reply:
Going directly to the newer commit ist fine with me.